deploy/lvm-operator.yaml

```diff
@@ -1497,6 +1497,7 @@ metadata:
   name: openebs-lvm-node-service
   labels:
     name: openebs-lvm-node
+  namespace: kube-system
```

Comment:
n_: push the `namespace` under the `name` field to be consistent with other objects.

Reply:
done.